pkg/service/backup_routine_handler.go

```diff
@@ -328,7 +342,7 @@ func (h *BackupRoutineHandler) runIncrementalBackupInternal(ctx context.Context,
 		return err
 	}
 
-	h.lastRun.incremental = now
+	h.lastRun.Incremental = &now
```

Comment:
Consider synchronizing.

Reply:
No backup of the same type for the same routine will run simultaneously, no need for lock